src/App.css

```diff
@@ -0,0 +1,12 @@
+*
```

Comment:
We can reset some browser styles here or we can create a separate file. It's your call.

Reply:
There's a better way to reset all styles, not just using *. And we'll have to decide on `content-box`. But it's good for now =)